features/filmStrip/components/native/styles/Styles.js

```diff
@@ -5,6 +5,15 @@ import { ColorPalette } from '../../../../base/styles';
 
 const styles = StyleSheet.create({
     /**
+     * Avatar style.
+     */
+
```

Comment:
There should be no empty lines between the JavaScript documentation comment and the entity it documents.